Package.swift

```diff
@@ -219,6 +219,7 @@ let SwiftFirebase =
                                 cxxSettings: [
                                   .define("INTERNAL_EXPERIMENTAL"),
                                   .define("_CRT_SECURE_NO_WARNINGS", .when(platforms: [.windows])),
+                                  .headerSearchPath("../../third_party/firebase-development/usr/include"),
```

Comment:
Hmm, why is this needed? `FireBaseUI` doesn't directly use the firebase APIs, it uses the Swift module only.

Reply:
Oof, that looks like something that we should expose differently. Yeah, I think that we should add that direct dependency here.